crates/miden-lib/asm/kernels/transaction/lib/account_delta.masm

```diff
@@ -99,6 +101,171 @@ export.compute_commitment
     end
 end
 
+#! Initializes the delta to the initial storage values in the account, by reading them from
+#! the advice inputs.
+#!
+#! This is only relevant for new accounts and only does actual work for storage maps, since
+#! value slots aren't explicitly tracked by the delta.
+#!
+#! Inputs:  []
+#! Outputs: []
+export.insert_new_account
+    exec.memory::get_native_account_initial_storage_slots_ptr
+    # => [init_slots_ptr]
```

Comment:
We expect that at this point initial storage slot are set to their initial values (i.e., `mem_copy_initial_storage_slots` has already been called) - right?